DashAI/back/dataloaders/params_schemas/jsondataloader.json

```
@@ -1,118 +1,129 @@
+        {
+          "error_msg": "",
+          "description": "Custom name to register your dataset, if left blank, the name of the uploaded file will be used.",
+          "type": "text",
+          "default": "",
+          "optional": null
+        }
+      ]
+    },
+    "data_key": {
+      "oneOf": [
+        {
+          "error_msg": "Write the name of the key that constains the data.",
+          "description": "The data has to be inside a key in your JSON files. Write the name of the key that constains the data.",
+          "type": "text",
+          "default": "data"
+        }
+      ]
+    },
+    "outputs_columns": {
+      "oneOf": [
+        {
+          "error_msg": "Need to specify the name(s) of the key(s) that defines the output columns of the data.",
+          "description": "You must specify which columns will be the output according to the task. By default the last column is selected.",
+          "type": "list_of_strings",
+          "default": []
         }
+      ]
     },
-    "splits": {
-        "display": "div",
-        "properties": {
-            "train_size": {
-                "oneOf": [
-                    {
-                        "error_msg": "The size of the training set must be between 0 and 1.",
-                        "description": "The training set contains the data to be used for training a model. Must be defined between 0 and 100% of the data.",
-                        "type": "float",
-                        "default": 0.7,
-                        "minimum": 0,
-                        "maximum": 1
-                    }
-                ]
-            },
-            "test_size": {
-                "oneOf": [
-                    {
-                        "error_msg": "The test set size must be between 0 and 1.",
-                        "description": "The test set contains the data that will be used to evaluate a model. Must be defined between 0 and 100% of the data.",
-                        "type": "float",
-                        "default": 0.2,
-                        "minimum": 0,
-                        "maximum": 1
-                    }
-                ]
-            },
-            "val_size": {
-                "oneOf": [
-                    {
-                        "error_msg": "The validation set size must be between 0 and 1.",
-                        "description": "The validation set contains the data to be used to validate a model. Must be defined between 0 and 100% of the data.",
-                        "type": "float",
-                        "default": 0.1,
-                        "minimum": 0,
-                        "maximum": 1
-                    }
-                ]
+    "splits_in_folders": {
+      "oneOf": [
+        {
+          "error_msg": "Need to select 'true' if you have folders that define the splits, otherwise 'false'.",
+          "description": "If your data has folders that define the splits select 'true', otherwise 'false'.",
```

"True allows you to indicate that the data are divided into "train", "test" and "validation" folders. False otherwise, indicates there are no previous divisions of the dataset."
```suggestion
          "description": "Option that allows you to indicate that the data are divided into "train", "test" and "validation" folders. Otherwise, it is assumed that there are no previous divisions of the dataset.",
```